Removed 'CA Communications' from getting-started.md (#123)
mozilla/www.ccadb.org: cas/getting-started.md

# Getting Started with the CCADB #

> When you [login](https://docs.google.com/document/d/1MuszDO2o-es_6-FwNCDWZ2TC979F8eil5bBQGKjaWRc/edit#heading=h.xpgou83dsc4m) to the CCADB you will see a row of tabs along the top of the window, next to the CCADB logo. The number of tabs that you see depends on the width of your window, and a 'More' tab will be shown when all of the tabs cannot be displayed. The tabs are: 'Home', 'My CA', 'CA Owners/Certificates', 'Cases', and 'Reports'.

Reviewer: ```suggestion
When you [log in](https://docs.google.com/document/d/1MuszDO2o-es_6-FwNCDWZ2TC979F8eil5bBQGKjaWRc/edit#heading=h.xpgou83dsc4m) to the CCADB you will see a row of tabs along the top of the window, next to the CCADB logo. The number of tabs that you see depends on the width of your window, and a 'More' tab will be shown when all of the tabs cannot be displayed. The tabs are: 'Home', 'My CA', 'CA Owners/Certificates', 'Cases', and 'Reports'.
```

> When you [login](https://docs.google.com/document/d/1MuszDO2o-es_6-FwNCDWZ2TC979F8eil5bBQGKjaWRc/edit#heading=h.xpgou83dsc4m) to the CCADB you will see a row of tabs along the top  of the window, next to the CCADB logo. The number of tabs that you see depends on the width of your window, and a 'More' tab will be shown when all of the tabs cannot be displayed. The tabs are: 'Home', 'My CA', 'CA Owners/Certificates', 'Cases', and 'Reports'.

Reviewer: ```suggestion
When you [log in](https://docs.google.com/document/d/1MuszDO2o-es_6-FwNCDWZ2TC979F8eil5bBQGKjaWRc/edit#heading=h.xpgou83dsc4m) to the CCADB, you will see a row of tabs along the top  of the window, next to the CCADB logo. The number of tabs you see depends on the width of your window, and a 'More' tab will be shown when all of the tabs cannot be displayed. The tabs are: 'Home', 'My CA', 'CA Owners/Certificates', 'Cases', and 'Reports'.
```

> **Home**: The home page is the initial page that you see when you login. It provides your CA's task list which is a set of reports that may be openned by clicking on their title.

Reviewer: ```suggestion
**Home**: The home page is the initial page that you see when you log in. It provides your CA's task list, a set of reports that may be opened by clicking on their title.
```

Note: The CCADB considers the terms "intermediate" and "subordinate" synonymous.

> **My CA**: Click on the 'My CA' tab to see your CA information that is in the CCADB (including Points of Contact), a hierarachy of root certiicates and intermediate certificates, and all cases, reports, and communications for your CA.

Reviewer: ```suggestion
**My CA**: Click on the 'My CA' tab to see your CA information in the CCADB (including Points of Contact), a hierarchy of root certificates and intermediate certificates, and all cases, reports, and communications for your CA.
```

> **CA Owners/Certificates**: Click on the 'CA Owners/Certificates' tab  (which may be under the 'More' tab) to see lists of certificates. 

Reviewer: ```suggestion
**CA Owners/Certificates**: Click on the 'CA Owners/Certificates' tab (which may be under the 'More' tab) to see lists of certificates. 
```

* Click on 'Recently Viewed' in the upper left corner. 

> * Select 'Community User's CA Owners/Root Certs' then click on the thumb tack icon to pin this view as your default for this tab.

Reviewer: ```suggestion
* Select 'Community User's CA Owners/Root Certs', then click on the thumbtack icon to pin this view as your default for this tab.
```

> **Cases**: Click on the 'Cases' tab (which may be under the 'More' tab) to see  lists of Cases.

Reviewer: ```suggestion
**Cases**: Click on the 'Cases' tab (which may be under the 'More' tab) to see lists of Cases.
```

* Click on 'Recently Viewed' in the upper left corner. 

> * Select 'My Cases' then click on the thumb tack icon to pin this view as your default for this tab.

Reviewer: ```suggestion
* Select 'My Cases', then click on the thumbtack icon to pin this view as your default for this tab.
```

    * '[Add/Update Root Request](updates)' cases are used for adding new root CA certificates or submitting annual updates.
    * '[Root Inclusion Request](inclusion)' cases are used for submitting requests for inclusion to Root Store Programs.

>     * '[Add/Update Contacts](contacts)' cases are used to add or udpate POC information. 

Reviewer: ```suggestion
    * '[Add/Update Contacts](contacts)' cases are used to add or update POC information. 
```

> **Reports**: Click on the 'Reports' tab (which may be under the 'More' tab), then click on 'All Folders' along the left column, and then click on 'CA Community Reports'.

Reviewer: ```suggestion
**Reports**: Click on the 'Reports' tab (which may be under the 'More' tab), then click 'All Folders' along the left column, and then click 'CA Community Reports'.
```
